daffodil-test-ibm1/src/test/resources/test-suite/ibm-contributed/dpaextdeltxt101-err.dfdl.xsd

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
```

Comment:
Add comment as to why this has to be a separate file, which is so we can cross test against IBM DFDL. IBM DFDL is stricter about errors in "other parts" of the schema than what a test is specifically executing. So schemas specifically intended for negative testing that produce SDE, have to be isolated from schema for tests that do not have SDEs.